src/app/shared/issue-tables/issue-tables.component.ts

```diff
@@ -0,0 +1,126 @@
+import { Component, Input, OnInit, ViewChild } from '@angular/core';
+import { Issue, STATUS } from '../../core/models/issue.model';
+import { PermissionService } from '../../core/services/permission.service';
+import { LabelService } from '../../core/services/label.service';
+import { UserService } from '../../core/services/user.service';
+import { GithubService } from '../../core/services/github.service';
+import { IssueService } from '../../core/services/issue.service';
+import { ErrorHandlingService } from '../../core/services/error-handling.service';
+import { finalize } from 'rxjs/operators';
+import { IssuesDataTable } from './IssuesDataTable';
+import { MatPaginator, MatSort } from '@angular/material';
+
+export enum ACTION_BUTTONS {
+  VIEW_IN_WEB,
+  MARK_AS_RESPONDED,
+  MARK_AS_PENDING,
+  RESPOND_TO_ISSUE,
+  FIX_ISSUE,
+  DELETE_ISSUE
+}
+
+@Component({
+  selector: 'app-issue-tables',
+  templateUrl: './issue-tables.component.html',
+  styleUrls: ['./issue-tables.component.css']
+})
+export class IssueTablesComponent implements OnInit {
+
+  @Input() issues: IssuesDataTable;
```

Comment:
Is it necessary to define `IssueDataTable` as an @Input since we are already creating a new instance of `IssuesDataTable` when `ngOnInit()`.

Reply:
Yes it can be removed. Was using it initially before I made the component generate the issues on init.